apps/journeys-admin/src/components/SignIn/RegisterPage/RegisterPage.tsx

```diff
@@ -72,7 +78,10 @@ export function RegisterPage({
     await signInWithEmailAndPassword(auth, email, password)
   }
 
-  async function handleCreateAccount(values, { setFieldError }): Promise<void> {
+  async function handleCreateAccount(
+    values: Values,
+    { setFieldError }
+  ): Promise<void> {
```

Comment:
```suggestion
  async function handleCreateAccount(values: values: InferType<typeof validationSchema>, { setFieldError }): Promise<void> {
```

```
import { InferType } from 'yup'
```

This will force the schema to stay in sync.